guides/typography.md

```diff
@@ -1,141 +1,143 @@
+| `display-1`     | `.mat-display-1`                 | Large, one-off header, usually at the top of the page (e.g. a hero header). |
+| `headline`      | `.mat-h1`, `.mat-headline`       | Section heading corresponding to the `<h1>` tag.                            |
+| `title`         | `.mat-h2`, `.mat-title`          | Section heading corresponding to the `<h2>` tag.                            |
+| `subheading-2`  | `.mat-h3`, `.mat-subheading-2`   | Section heading corresponding to the `<h3>` tag.                            |
+| `subheading-1`  | `.mat-h4`, `.mat-subheading-1`   | Section heading corresponding to the `<h4>` tag.                            |
+| `body-1`        | `.mat-body`, `.mat-body-1`       | Base body text.                                                             |
+| `body-2`        | `.mat-body-strong`, `.mat-body-2`| Bolder body text.                                                           |
+| `caption`       | `.mat-small`, `.mat-caption`     | Smaller body and hint text.                                                 |
+| `button`        | None. Used only in components.   | Buttons and anchors.                                                        |
+| `input`         | None. Used only in components.   | Form input fields.                                                          |
+
+
+The typography levels are collected into a typography config which is used to generate the CSS.
+
+### Usage
+To get started, you first include the `Roboto` font with the 300, 400 and 500 weights.
+You can host it yourself or include it from [Google Fonts][2]:
+
+```html
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+```
+
+Now you can add the appropriate CSS classes to the elements that you want to style:
+
+```html
+<h1 class="mat-display-1">Jackdaws love my big sphinx of quartz.</h1>
+<h2 class="mat-h2">The quick brown fox jumps over the lazy dog.</h2>
+```
+
+By default, Angular Material doesn't apply any global CSS. To apply the library's typographic styles
+more broadly, you can take advantage of the `mat-typography` CSS class. This class will style all
+descendant native elements.
+
+```html
+<!-- By default, Angular Material applies no global styles to native elements. -->
+<h1>This header is unstyled</h1>
+
+<!-- Applying the mat-tyography class adds styles for native elements. -->
+<section class="mat-typography">
+  <h1>This header will be styled</h1>
+</section>
+```
+
+### Customization
+Typography customization is an extension of Angular Material's Sass-based theming. Similar to
+creating a custom theme, you can create a custom **typography configuration**.
+
+```scss
+@import '~@angular/material/theming';
+
+// Define a custom typography config that overrides the font-family as well as the
+// `headlines` and `body-1` levels.
+$custom-typography-theme: (
+    typography: mat-typography-config(
```

Comment:
I moved a footnote that was the the top for some reason which threw off the diff alog. This is what I actually changed. I wrapped the `mat-typography-config` value in a theme map `(typography: ...)` and changed the variable to `$custom-typograpy-theme`